helm/vitalam-identity-service/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "vitalam-identity-service.fullname" . }}-config
+  labels:
+    {{- include "vitalam-identity-service.labels" . | nindent 4 }}
+data:
+  port: {{ .Values.config.port | quote }}
+  logLevel: {{ .Values.config.logLevel }}
+  dbHost: {{ include "vitalam-identity-service.fullname" . }}
+  dbPort: {{ .Values.config.dbPort | quote }}
+  dbName: {{ .Values.config.dbName }}
```

Comment:
We need to be have some of the `ENVARS` able to be put into this configmap

Reply:
Agreed by default all the envs should be configurable by the helm chart

Reply:
done